lib/stat_tracker.rb

```diff
@@ -1,99 +1,104 @@
-require 'csv'
-require_relative 'team_stats'
-require_relative 'game_teams_stats'
-require_relative 'game_stats'
 require_relative 'game'
+require_relative 'team'
+require_relative 'game_team'
+require_relative 'game_stats'
+require_relative 'league_stats'
+# require_relative 'season_stats'
+# require_relative 'team_stats'
+require_relative './modules/data_loadable'
 
 class StatTracker
+  attr_reader :games, :teams, :game_teams, :game_stats, :league_stats
   include DataLoadable
-  attr_reader :games_path, :team_path, :game_teams_path
 
   def self.from_csv(locations)
     StatTracker.new(locations)
   end
 
   def initialize(locations)
```

Comment:
I like how the file structure has been reorganized

Reply:
I like how the file structure has been reorganized ™